resources/features/Sampling.feature

```diff
@@ -1,75 +1,165 @@
+        And a Strata of selection type Prn-Sampling from Stratification Properties file:
+            | inqcode | cell_no | cell_desc | seltype | lower_class | upper_class | lower_size | upper_size | prn_start   | no_reqd |
+            | 687     | 5816    | Sample    | P       | 45111       | 45190       | 50         | 9          | 0.539298879 | 3       |
+            | 687     | 5811    | Admin     | U       | 45111       | 45190       | 0          | 9          | 0.000000000 | 0       |
+            | 687     | 5812    | Admin     | U       | 45111       | 45190       | 10         | 49         | 0.000000000 | 0       |
         When a <language> Sample is created from a Stratified Frame
-        Then a Sample DataFrame is returned and exported to CSV with the inclusion of stratas with outbound Sample Size parameter
+        Then a Sample is returned and exported to CSV with the inclusion of stratas with outbound Sample Size parameter
+            | ern        | entref     | name                          | tradingstyle | address1                 | address2       | address3    | address4                | address5 | postcode | legalstatus | sic07 | paye_empees | paye_jobs | ent_turnover | std_turnover | grp_turnover | cntd_turnover | app_turnover | prn         | cell_no |
+            | 1100000002 | 9906000045 | BUEADLIING SOLUTIONS LTD      |              | 1 HAZELWOOD LANE         | ABBOTS LANGLEY |             |                         |          | WD5 0HA  | 3           | 45190 | 1           | 0         | 100          | 100          | 0            | 0             | 0            | 0.63848639  | 5816    |
+            | 1100000001 | 9906000015 | &EAGBBROWN                    |              | 1 HAWRIDGE HILL COTTAGES | THE VALE       | HAWRIDGE    | CHESHAM BUCKINGHAMSHIRE |          | HP5 3NU  | 1           | 45112 | 1           | 1         | 73           | 73           | 0            | 0             | 0            | 0.109636832 | 5816    |
+            | 1100000007 | 9906000275 | IBANOCTRACTS UK LTD           |              | 1 GLYNDE PLACE           | HORSHAM        | WEST SUSSEX |                         |          | RH12 1NZ | 1           | 46120 | 2           | 2         | 287          | 287          | 0            | 0             | 0            | 0.155647458 | 5816    |
 
         @JVM
         Examples:
-        | language | input_path                                 | strat_properties_path                          | output_path                                         |
-        | Scala    | ./resources/inputs/sampling/frame_data.csv | ./resources/inputs/sampling/out_bounds_prn.csv | ./resources/outputs/sampling/sample_data/out_bounds |
+        | language |
+        | Scala    |
+
+
+    Scenario Outline: Stratified Frame file cannot be found in given Frame directory and an exception is thrown
```

Comment:
Is the test correct here given that we are talking about a missing input frame, shouldn't the 'Given' be blank?